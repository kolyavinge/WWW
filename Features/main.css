.d1 {
    background-color: #d4dbff;
    width: 500px;
    height: 500px;
    display: flex;
    align-items: center;
}

.d2 {
    background-color: #d4ffd9;
    width: 50px;
    height: 50px;
    margin: 0 auto;
}
