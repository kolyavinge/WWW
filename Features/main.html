<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title></title>
    <link href="main.css" rel="stylesheet" />
</head>
<body>
    <div class="d1">
        <div class="d2">1</div>
    </div>
</body>
</html>
