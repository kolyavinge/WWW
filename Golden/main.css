* {
    padding: 0;
    margin: 0;
    /*border: 1px solid red;*/
}

.clearfix:after {
    content: '';
    display: table;
    width: 100%;
    clear: both;
}

header {
    height: 100vh;
    background: url("img/first-bg_1.jpg") no-repeat center top / cover;
}

.container {
    width: 940px;
    margin: 0 auto;
    font-family: Arial, sans-serif;
}

.logo {
    margin-top: 35px;
    float: left;
}

.menu {
    margin-top: 45px;
    float: right;
}

    .menu ul li {
        list-style-type: none;
        display: inline;
        margin-right: 40px;
        text-transform: uppercase;
        font-size: 14pt;
    }

    .menu a {
        color: #fff;
        text-decoration: none;
    }

.welcome {
    margin-top: 277px;
    text-transform: uppercase;
    color: #fff;
    font-size: 28pt;
    text-align: center;
}

.button {
    display: block;
    margin: 100px auto 0 auto;
    color: #494642;
    background-color: #FED136;
    border-radius: 3px;
    width: 240px;
    line-height: 65px;
    text-decoration: none;
    text-transform: uppercase;
    text-align: center;
    font-weight: bold;
}

.services-titles {
    padding-top: 120px;
}

    .services-titles h2 {
        text-transform: uppercase;
        color: #000;
        font-size: 28pt;
        text-align: center;
    }

    .services-titles p {
        color: #9D9D9D;
        text-align: center;
        font-size: 14pt;
    }

.services-types {
    margin-top: 80px;
    padding-bottom: 125px;
}

.services-type {
    float: left;
    width: 33%;
}

    .services-type img {
        display: block;
        margin: 0 auto;
    }

    .services-type p {
        color: #9D9D9D;
        text-align: center;
        font-size: 14pt;
    }

#portfolio {
    background-color: #F7F7F7;
}

.portfolio-titles {
    padding-top: 80px;
}

    .portfolio-titles h2 {
        text-transform: uppercase;
        color: #000;
        font-size: 28pt;
        text-align: center;
    }

    .portfolio-titles p {
        color: #9D9D9D;
        text-align: center;
        font-size: 14pt;
    }

.portfolio-types {
    margin-top: 80px;
    padding-bottom: 125px;
}

.portfolio-type {
    float: left;
    width: 33%;
}

    .portfolio-type img {
        display: block;
        margin: 0 auto;
    }

.about-titles {
    padding-top: 80px;
}

    .about-titles h2 {
        text-transform: uppercase;
        color: #000;
        font-size: 28pt;
        text-align: center;
    }

    .about-titles p {
        color: #9D9D9D;
        text-align: center;
        font-size: 14pt;
    }

.about {
    margin-top: 90px;
    margin-bottom: 90px;
}

    .about img {
        display: block;
        float: left;
        margin-left: 50px;
        width: 183px;
    }

.about-text-left {
    float: left;
    width: 342px;
    text-align: right;
}

.about-text-right {
    float: left;
    width: 342px;
    text-align: right;
}
