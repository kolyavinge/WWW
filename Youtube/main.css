* {
	padding: 0;
	margin: 0;
}

.container {
}

header {
	background-color: #ffe4e4;
	height: 52px;
}

nav {
	background-color: #e4ebff;
	float: left;
	width: 256px;
	height: 100%;
}

aside {
	background-color: #e4ffe7;
	float: left;
	width: 500px;
	height: 100%;
}
