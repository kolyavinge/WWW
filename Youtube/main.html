<html xmlns="http://www.w3.org/1999/xhtml">
<head>
	<title></title>
	<link href="main.css" rel="stylesheet" />
</head>
<body>
	<header>
		<div class="container">

		</div>
	</header>
	<nav>
		<div class="container">

		</div>
	</nav>
	<aside>
		<div class="container">

		</div>
	</aside>
</body>
</html>
