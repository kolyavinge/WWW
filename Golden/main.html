<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title></title>
    <link href="main.css" rel="stylesheet" />
</head>
<body>
    <header>
        <div class="container">
            <div class="clearfix">
                <div class="logo">
                    <img src="img/logo.png" alt="Logo" />
                </div>
                <div class="menu">
                    <ul>
                        <li><a href="#">Home</a></li>
                        <li><a href="#">Services</a></li>
                        <li><a href="#">Portfolio</a></li>
                        <li><a href="#">About</a></li>
                        <li><a href="#">Contact</a></li>
                    </ul>
                </div>
            </div>
            <div class="welcome">
                <p>Welcome to our studio!</p>
                <h1>It's nice to meet you</h1>
            </div>
            <a class="button" href="#" alt="Tell me more">Tell me more</a>
        </div>
    </header>
    <section>
        <div class="container">
            <div class="services-titles">
                <h2>Services</h2>
                <p>Proin iaculis ...</p>
            </div>
            <div class="services-types clearfix">
                <div class="services-type">
                    <img src="img/icon1.png" alt="E-Commerce" />
                    <p>text</p>
                </div>
                <div class="services-type">
                    <img src="img/icon2.png" alt="Responsive web" />
                    <p>text</p>
                </div>
                <div class="services-type">
                    <img src="img/icon3.png" alt="Web security" />
                    <p>text</p>
                </div>
            </div>
        </div>
    </section>
    <section id="portfolio">
        <div class="container">
            <div class="portfolio-titles">
                <h2>Our portfolio</h2>
                <p>Proin iaculis ...</p>
            </div>
            <div class="portfolio-types clearfix">
                <div class="portfolio-type">
                    <img src="img/img1.jpg" alt="E-Commerce" />
                </div>
                <div class="portfolio-type">
                    <img src="img/img2.jpg" alt="Responsive web" />
                </div>
                <div class="portfolio-type">
                    <img src="img/img3.jpg" alt="Web security" />
                </div>
            </div>
        </div>
    </section>
    <section>
        <div class="container">
            <div class="about-titles">
                <h2>About us</h2>
                <p>Proin iaculis ...</p>
            </div>
            <div class="about clearfix">
                <div class="about-text-left">
                    <h3>July 2010</h3>
                    <h3>Our humble beginnings</h3>
                    <p>Text</p>
                </div>
                <img src="img/about1.jpg" alt="Alternate Text" />
            </div>
            <div class="about clearfix">
                <div class="about-text-left">&nbsp;</div>
                <img src="img/about2.jpg" alt="Alternate Text" />
				<div class="about-text-right">&nbsp;</div>
            </div>
            <div class="about clearfix">
                <div class="about-text-left">&nbsp;</div>
                <img src="img/about3.jpg" alt="Alternate Text" />
				<div class="about-text-right">&nbsp;</div>
            </div>
            <div class="about clearfix">
                <div class="about-text-left">&nbsp;</div>
                <img src="img/about4.jpg" alt="Alternate Text" />
				<div class="about-text-right">&nbsp;</div>
            </div>
        </div>
    </section>
</body>
</html>
